cmd/flags/admin.go

```diff
@@ -107,7 +107,11 @@ func (s *AdminServer) serveWithListener(l net.Listener) {
 	version.RegisterHandler(s.mux, s.logger)
 	s.registerPprofHandlers()
 	recoveryHandler := recoveryhandler.NewRecoveryHandler(s.logger, true)
-	s.server = &http.Server{Handler: recoveryHandler(s.mux)}
+	s.server = &http.Server{
+		Handler:  recoveryHandler(s.mux),
+		ErrorLog: zap.NewStdLog(s.logger),
```

Comment:
this is incorrect because the errors will be logged at Info. Need to use `NewStdLogAt(logger, zapcore.ErrorLevel)`